core/src/com/unciv/ui/tutorials/TutorialMiner.kt

```diff
@@ -0,0 +1,41 @@
+package com.unciv.ui.tutorials
+
+import com.badlogic.gdx.Gdx
+import com.badlogic.gdx.utils.Array
+import com.unciv.UncivGame
+import com.unciv.models.Tutorial
+
+interface TutorialMiner {
+
+    fun getAllTutorials(language: String): Map<Tutorial, List<String>>?
+    fun getCivilopediaTutorials(language: String): Map<Tutorial, List<String>>?
+    fun getTutorial(tutorial: Tutorial, language: String): List<String>?
+
+    class Impl : TutorialMiner {
+
+        companion object {
+            private const val TUTORIALS_PATH = "jsons/Tutorials/Tutorials_%s.json"
+        }
+
+        override fun getAllTutorials(language: String): Map<Tutorial, List<String>>? {
+            val path = TUTORIALS_PATH.format(language)
+            if (!Gdx.files.internal(path).exists()) return null
+
+            // ...Yes. Disgusting. I wish I didn't have to do this.
+            val x = LinkedHashMap<String, Array<Array<String>>>()
+            val tutorials: LinkedHashMap<String, Array<Array<String>>> = UncivGame.Current.ruleset.getFromJson(x.javaClass, path)
+
+            val tutorialMap = mutableMapOf<Tutorial, List<String>>()
+            tutorials.forEach { t ->
+                tutorialMap[Tutorial.findByName(t.key)!!] = t.value.map { it.joinToString("\n") }
+            }
+            return tutorialMap
+        }
+
+        override fun getCivilopediaTutorials(language: String): Map<Tutorial, List<String>>? =
+                getAllTutorials(UncivGame.Current.settings.language)?.filter { !it.key.name.startsWith("_") }
+
+        override fun getTutorial(tutorial: Tutorial, language: String): List<String>? =
```

Comment:
Aha, so this is the source of all evil. Once this cannot return null, all the rest of the chain will be fixed accordingly. 
Basically, if you can't get ANY tutorials AT ALL, even from English, return an empty list.